ibm_cloud_sdk_core/authenticators/container_authenticator.py

```diff
@@ -66,6 +66,7 @@ class ContainerAuthenticator(IAMRequestBasedAuthenticator):
 
     def __init__(
         self,
+        *,
```

Comment:
Arguments before the asterisk are positional argument, so they can be passed like `call_function(1, 2)`. Arguments after the `*` become keyword only, so they cannot be used in the previous way, only like this: `call_function(a=1, b=2)`. It's not advised to have too many positional arguments because for example to specify the last one, you have to specify all the previous one too. Since they are used by their position not their keyword.
TLDR; there is a [new `pylint` rule](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/too-many-positional-arguments.html) that complains about this.
Yes, this changes the behavior a little, but I don't think this would be a breaking change. If we start getting negative feedback, we can restore the original behavior pretty easily and exclude the affected functions/methods from the linter.